<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Join</class>
 <widget class="QWidget" name="Join">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ico/schat);
background-color: rgb(255, 255, 255,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="back_0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>395</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(240, 240, 240, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="back_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>25</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(63, 188, 242);</string>
   </property>
   <property name="text">
    <string>SChat</string>
   </property>
  </widget>
  <widget class="QLabel" name="avatar">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>90</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ico/unknown);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>0</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255,0);
color: rgb(255, 255, 255);
border:0;
}
QPushButton:hover{
background-color: rgb(255, 0, 0, 100);
color: rgb(255, 255, 255);
border:0;
}
QPushButton:pressed{
background-color: rgb(255, 0, 0, 200);
color: rgb(255, 255, 255);
border:0;
}</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="min">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>0</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255,0);
color: rgb(255, 255, 255);
border:0;
}
QPushButton:hover{
background-color: rgb(255, 255, 255, 100);
color: rgb(255, 255, 255);
border:0;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255, 200);
color: rgb(255, 255, 255);
border:0;
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="back_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>540</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 100);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="container" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>541</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="container_1">
     <item>
      <widget class="QLineEdit" name="nick">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255,0);
outline: none;
border: none;
border-bottom: 2px solid rgb(100, 100, 100);
selection-background-color: rgb(63, 188, 242);
}
QLineEdit:hover{
border-bottom: 2px solid rgb(20, 20, 20);
}
QLineEdit:focus{
border-bottom: 2px solid rgb(63, 188, 242);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>昵称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="room">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255,0);
outline: none;
border: none;
border-bottom: 2px solid rgb(100, 100, 100);
selection-background-color: rgb(63, 188, 242);
}
QLineEdit:hover{
border-bottom: 2px solid rgb(20, 20, 20);
}
QLineEdit:focus{
border-bottom: 2px solid rgb(63, 188, 242);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>频道</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="auto_login">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>自动登陆</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="rember_room">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>记住房间</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="login">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(63, 188, 242);
color: #ffffff;
border:0;
}
QPushButton:hover{
background-color: rgb(123, 207, 242);
color: #ffffff;
border:0;
}
QPushButton:pressed{
background-color: rgb(33, 180, 242);
color: #ffffff;
}</string>
       </property>
       <property name="text">
        <string>登陆</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="onLogin" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>159</y>
     <width>521</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>491</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>27</y>
      <width>431</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(63, 188, 242, 100);</string>
    </property>
    <property name="text">
     <string>▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰
正在登陆
Now Login ...
▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰▰</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="icon_back">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>88</y>
     <width>94</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="set">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255,0);

border:0;
}
QPushButton:hover{
background-color: rgb(255, 255, 255, 100);
color: rgb(255, 255, 255);
border:0;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255, 200);
color: rgb(255, 255, 255);
border:0;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set_label">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>14</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ico/set_btn);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>back_2</zorder>
  <zorder>back_0</zorder>
  <zorder>back_1</zorder>
  <zorder>icon</zorder>
  <zorder>title</zorder>
  <zorder>close</zorder>
  <zorder>min</zorder>
  <zorder>icon_back</zorder>
  <zorder>avatar</zorder>
  <zorder>onLogin</zorder>
  <zorder>container</zorder>
  <zorder>set_label</zorder>
  <zorder>set</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>Join</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min</sender>
   <signal>clicked()</signal>
   <receiver>Join</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
